### DISCUSSION TITLE
Load env variables in workbench

### DISCUSSION THREAD

**Discussion by xusimr97** [orchestral/testbench#419] [Q&A]
I'm trying to load environment variables from my .env file, but I'm encountering issues. The only way it works is by hardcoding the variables in the env attribute within testbench.yaml.

However, I prefer not to hardcode the environment variables in that file. Is there a way to automatically load the environment variables from the .env file located in my package's workbench folder?

I've also attempted loading the variables through a service provider, but that approach didn't work either.

**Comment by crynobone** [comment]
add the environment file to `workbench/.env` and you can opt for `.env.example` or `.env.dist` too
